articles/cosmos-db/changefeed-ecommerce-solution.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Use Azure Cosmos DB change feed to visualize real-time data analytics | Microsoft Docs
+title: Use Azure #CosmosDB change feed to visualize real-time data analytics | Microsoft Docs
```

Comment:
@serenadavis #CosmosDB handle is only applicable for blogs, it should be be used in docs title.